server/database/queries/deleteProduct.js

```diff
@@ -0,0 +1,8 @@
+const connection = require('../connection');
+
+const deleteProduct = (productId) => connection.query({
+  text: 'delete from products where id = $1',
```

Comment:
when I try to delete a product with id =1, I got this error
```
update or delete on table "products" violates foreign key constraint "orders_product_id_fkey" on table "orders"
```
please, handle this case when the product has relation to other tables in the database

Reply:
done